### PR TITLE
[FINE] Set the location property of managed disks to ensure disk saving

### PR DESCRIPTION
**Pull request by juliancheal**
Set the location property of managed disks to ensure disk saving

Backport of https://github.com/ManageIQ/manageiq-providers-azure/pull/143
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1505951

### CODE REVIEW

**Comment by juliancheal** [comment]
@miq-bot assign @simaishi

**Comment by juliancheal** [comment]
@miq-bot add_label fine/yes

**Comment by juliancheal** [comment]
Sorry I think I made a bit of a mess of this when trying to get the specs correct. /cc @simaishi

File: spec/models/manageiq/providers/azure/cloud_manager/refresher_spec.rb [hunk 1]
```diff
@@ -17,6 +17,7 @@
     @ip_address     = '13.82.180.158'  # This will change if you had to restart the @device_name.
     @mismatch_ip    = '13.82.224.98'   # This will change if you had to restart the 'miqmismatch' VM.
     @managed_disk   = "miqazure-linux-managed_OsDisk_1_7b2bdf790a7d4379ace2846d307730cd"
```
Comment (bronaghs) [inline spec/models/manageiq/providers/azure/cloud_manager/refresher_spec.rb]: @juliancheal - this variable name became `@managed_os_disk`

**Comment by juliancheal** [comment]
Ok I think we're all good now. Sorry about that :) /cc @bronaghs @simaishi

**Pull request merged**